### DISCUSSION TITLE
Haven't received my students benefits.

### DISCUSSION THREAD

**Discussion by SmokeTopusMan** [community/community#130812] [GitHub Education]
### Hi everyone,

hi, it's been 3 days and i still haven't gotten my student's account benefits. i realy need them to continue my work here, could you help me resolve this issue?
thanks in advanced

**Comment by emansarahafi** [comment]
Hello, sometimes it takes longer for the benefits to be processed, some users receive them within 3 days but it can take a while during long periods. I suggest you reach out to someone from the GitHub Education staff who could fix it for you by opening a ticket [here](https://support.github.com/contact/account), selecting Other, and explaining your situation.
